weblogic-azure-aks/src/main/arm/scripts/genDomainConfig.sh

```diff
@@ -5,24 +5,11 @@ export script="${BASH_SOURCE[0]}"
 export scriptDir="$(cd "$(dirname "${script}")" && pwd)"
 
 export filePath=$1
-export replicas=$2
-export wlsCPU=$3
-export wlsDomainUID=$4
-export wlsDomainName=$5
-export wlsImagePath=$6
-export wlsMemory=$7
-export wlsManagedPrefix=$8
-export enableSSL=${9}
-export enablePV=${10}
-export enableAdminT3Tunneling=${11}
-export enableClusterT3Tunneling=${12}
-export t3AdminPort=${13}
-export t3ClusterPort=${14}
-export clusterName=${15}
-export javaOptions=${16}
-
-export adminServiceUrl="${wlsDomainUID}-admin-server.${wlsDomainUID}-ns.svc.cluster.local"
-export clusterServiceUrl="${wlsDomainUID}-cluster-${clusterName}.${wlsDomainUID}-ns.svc.cluster.local"
+export wlsImagePath=$2
```

Comment:
This is a child script, called in different locations. Keep the positional variable here